app/controllers/stops_controller.rb

```diff
@@ -29,4 +29,10 @@ def stop_params
     params.require(:stop).permit(:name)
   end
 
+  def authorize_user
+    if !user_signed_in? || !current_user.admin?
+      raise ActionController::RoutingError.new("Not Found")
+    end
+  end
```

Comment:
All of this looks good....but you need to actually CALL your authorize user method! That way we actually use your good looking code. 
Double check the reading on this, but maybe you can call if for the `:new` action. 

Once that is in, you'll need a quick capybara test for that `new` page (or any other page that authorizes user) to ensure that unsigned in users cant navigate to the new page.